### PR TITLE
[zh-cn] Sync 2 files in tasks/inject-data-application/

### PR DESCRIPTION
**Pull request by yanrongshi**
Sync upstream
```yaml
content/zh-cn/docs/tasks/inject-data-application/downward-api-volume-expose-pod-information.md
content/zh-cn/docs/tasks/inject-data-application/environment-variable-expose-pod-information.md
```

### CODE REVIEW

**Comment by windsonsea** [comment]
/retitle [zh] Sync 2 files in tasks/inject-data-application/

File: content/zh-cn/docs/tasks/inject-data-application/environment-variable-expose-pod-information.md [hunk 1]
```diff
@@ -236,16 +235,29 @@ The output shows the values of selected environment variables:
...
+* Read [Defining Environment Variables for a Container](/docs/tasks/inject-data-application/define-environment-variable-container/)
+* Read the [`spec`](/docs/reference/kubernetes-api/workload-resources/pod-v1/#PodSpec)
+  API definition for Pod. This includes the definition of Container (part of Pod).
+* Read the list of [available fields](/docs/concepts/workloads/pods/downward-api/#available-fields) that you
+  can expose using the downward API.
+-->
+* 阅读[给容器定义环境变量](/zh-cn/docs/tasks/inject-data-application/define-environment-variable-container/)
+* 阅读 Pod 的 [`spec`](/docs/reference/kubernetes-api/workload-resources/pod-v1/#PodSpec)
+  API 包括容器（Pod 的一部分）的定义。
+* 阅读可以使用 downward API 公开的[可用字段](/docs/concepts/workloads/pods/downward-api/#available-fields)列表。
```
Comment (Sea-n) [inline content/zh-cn/docs/tasks/inject-data-application/environment-variable-expose-pod-information.md:247]:
```suggestion
* 阅读可以使用 downward API 公开的[可用字段](/zh-cn/docs/concepts/workloads/pods/downward-api/#available-fields)列表。
```

File: content/zh-cn/docs/tasks/inject-data-application/environment-variable-expose-pod-information.md [hunk 2]
```diff
@@ -236,16 +235,29 @@ The output shows the values of selected environment variables:
...
 <!--
-* [Defining Environment Variables for a Container](/docs/tasks/inject-data-application/define-environment-variable-container/)
+* Read [Defining Environment Variables for a Container](/docs/tasks/inject-data-application/define-environment-variable-container/)
+* Read the [`spec`](/docs/reference/kubernetes-api/workload-resources/pod-v1/#PodSpec)
+  API definition for Pod. This includes the definition of Container (part of Pod).
+* Read the list of [available fields](/docs/concepts/workloads/pods/downward-api/#available-fields) that you
+  can expose using the downward API.
+-->
+* 阅读[给容器定义环境变量](/zh-cn/docs/tasks/inject-data-application/define-environment-variable-container/)
+* 阅读 Pod 的 [`spec`](/docs/reference/kubernetes-api/workload-resources/pod-v1/#PodSpec)
```
Comment (Sea-n) [inline content/zh-cn/docs/tasks/inject-data-application/environment-variable-expose-pod-information.md:245]:
```suggestion
* 阅读 Pod 的 [`spec`](/zh-cn/docs/reference/kubernetes-api/workload-resources/pod-v1/#PodSpec)
```

**Review by Sea-n — COMMENTED** [review]
/approve

**Comment by windsonsea** [comment]
/lgtm

Thanks.

**Pull request merged**